helm/charts/determined/templates/master-config.yaml

```diff
@@ -118,6 +118,8 @@ data:
       fluent:
         {{- if .Values.fluent.image }}
         image: {{ .Values.fluent.image }}
+        uid: {{ .Values.fluent.uid }}
```

Comment:
this is under `if .Values.fluent.image` which doesn't seem right because it won't allow setting uid/gid without image. 
since contents of Values.fluent are 1:1 with the master config, can we embed it here directly, without going over the fields individually? i.e.
```
{{- if .Values.fluent }}
  {{- toYaml .Values.fluent | nindent 4}}
{{ end }}
```

(I may be misjudging the nindent value)

Reply:
Whoops, good catch and good idea. Changed.